src/guide/advanced/running-iroha-on-bare-metal.md

```diff
@@ -49,83 +69,103 @@ export IROHA2_CONFIG_PATH="$(pwd)/Git/iroha/configs/peer/config.json"
 
 For extra convenience, you could add the Iroha 2 `target` directory to your
 `PATH`. This means that instead of having to specify the full path to the
-executable `iroha` you can instead type `iroha` directly into your command
-line. ::: tip
-
-Don’t forget to replace `debug` with `release` when you’re ready to deploy
-in the real world.
-
-:::
+executable `iroha`, you can instead type `iroha` directly into your command
+line.
 
 ```bash
 export PATH="$PATH:$(pwd)/Git/iroha/target/debug"
 ```
 
-This way you can run `iroha` from any directory, without having to worry
+::: tip
+
+Don't forget to replace `debug` with `release` when you're ready to deploy
+in the real world.
+
+:::
+
+This way you can run `iroha` from any directory without having to worry
 about configuration paths and/or specifying the full path to the Iroha
 executable.
 
-The instructions here should probably be made permanent, which you can do
-by adding the following environment variables to your startup shell. On
-older Linux systems, you copy and paste everything except `cd ~` to
-`~/.bashrc`. On macs running Mac OS X 10.6 and later, as well as some Linux
+It's a good idea to make these instructions permanent, which you can do by
+adding the environment variables to your startup shell.
+
+::: details Save the instructions to the startup shell
+
+On older Linux systems, you copy and paste the instructions (without the
+`cd ~`) to `~/.bashrc`. On Mac OS X 10.6 and later, as well as some Linux
 systems, you want to add the same lines to `~/.zshrc`.
 
+Copy these instructions to the specified files (replace `debug` with
+`release` when you're ready to deploy):
+
+```bash
+export IROHA2_GENESIS_PATH="$(pwd)/Git/iroha/configs/peer/genesis.json"
+export IROHA2_CONFIG_PATH="$(pwd)/Git/iroha/configs/peer/config.json"
+export PATH="$PATH:$(pwd)/Git/iroha/target/debug"
+```
+
+:::
+
 ::: tip
 
-This process is almost universally unreliable and messy, and it’s likely
-that your system is special in that it breaks some of our assumptions. If
-the above optional steps didn’t work for you, you can keep working in the
-`~/Git/iroha/configs/peer/` folder, and run iroha by doing
-`~/Git/iroha/target/debug/iroha`. This makes the command-line a little
-harder to read, which is why we recommend settin up your environment first.
+This process is almost universally unreliable and messy, and it's likely
+that your system is special in that it breaks some of our assumptions.
+
+If the above optional steps didn't work for you, you can keep working in
+the `~/Git/iroha/configs/peer/` folder, and run Iroha via
+`~/Git/iroha/target/debug/iroha`.
+
+This makes the command-line a little harder to read, which is why we
+recommend setting up your environment first.
 
 :::
 
 ::: info Note
 
-The tutorial assumes that you’re running either Linux, Mac OS X, or Windows
+The tutorial assumes that you're running either Linux, Mac OS X, or Windows
```

Comment:
Please revert back. It's rendered more correctly with the proper quotation mark symbol.

Reply:
done